### ISSUE TITLE
Accessibility audit: make project links easier to click on

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#3923]
From our accessibility audit:

> The individual project elements under the "trending projects" and "new release" headings are only
clickable if you hit the link element inside of them. Small clickable areas can be really hard to access
for a user with motor impairment. To fix this specific issue, you can make the whole box clickable or at
least all the text and the icon. If you do make the whole box clickable, be sure to leave enough space
between the boxes so that users are able to scroll on a mobile phone.

This affects:

- The homepage
- The search results page

**Comment by rixx** [comment]
This issue should be completed and can be closed.
